src/sagemaker/jumpstart/constants.py

```diff
@@ -149,3 +151,10 @@ class VariableScope(str, Enum):
 
     CONTAINER = "container"
     ALGORITHM = "algorithm"
+
+
+class JumpStartTag(str, Enum):
+    """Enum class for tags to apply to JumpStart models."""
+
+    INFERENCE_MODEL_URI = "jumpstart-inference-model-uri"
+    INFERENCE_SCRIPT_URI = "jumpstart-inference-script-uri"
```

Comment:
could you use something like: `aws-jumpstart-` or `aws-smjs-` prefix to make it clear this was added by AWS?

System tags look like `aws:` so I would like to be close to this, even though these are not system tags (see [tagging](https://docs.aws.amazon.com/general/latest/gr/aws_tagging.html)).